netlify.toml

```diff
@@ -1,4 +1,4 @@
 [[redirects]]
   from = "/"
-  to = "/1/bridge"
+  to = "/example/bridge"
```

Comment:
Should we default to one of the 2 daos rather than a non-existent one?

Reply:
now defaults to `pool-together`